chromadb/__init__.py

```diff
@@ -135,12 +139,16 @@ def PersistentClient(
     settings.persist_directory = path
     settings.is_persistent = True
 
+    # https://github.com/chroma-core/chroma/issues/1573
+    tenant = str(tenant)
+    database = str(database)
+
     return ClientCreator(tenant=tenant, database=database, settings=settings)
 
 
 def HttpClient(
     host: str = "localhost",
-    port: str = "8000",
+    port: int = 8000,
```

Comment:
what will happen if the user currently is passing a string? just a type error, right?

Reply:
Users can pass whatever they want -- these are just type hints which users can choose to statically check or not. If a user is currently passing a string we'll turn it into an int with the `port = int(port)` line below. This will not break anyone.